contracts/lazy_mint/ERC20/LazyMintERC20.sol

```diff
@@ -147,6 +147,33 @@ contract LazyMintERC20 is ILazyMintERC20, Coin, ReentrancyGuard {
 
     //      =====   Getter functions  =====
 
+    /// @dev Returns whether an address has claimed tokens during a claim condition.
+    function canClaim(
```

Comment:
can we just expose `verifyClaim` public? right now it's duplicating code basically
sdk can try catch and return meaningful messages